pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1271,6 +1271,20 @@ self: super: {
   # Created upstream PR @ https://github.com/ghcjs/jsaddle/pull/119
   jsaddle-webkit2gtk = appendPatch super.jsaddle-webkit2gtk ./patches/jsaddle-webkit2gtk.patch;
 
+  # 2021-05-12: gi-gdkpixbuf_2_0_26 fix
+  # https://github.com/taffybar/gtk-sni-tray/pull/25
+  gtk-sni-tray = markUnbroken (appendPatch super.gtk-sni-tray (pkgs.fetchpatch {
```

Comment:
Please don't use `markUnbroken` in nixpkgs itself, instead remove the packages in question from `pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml` and run `./maintainers/scripts/haskell/regenerate-hackage-packages.sh`.

Reply:
OK thanks, fixed.